lib/googleauth/user_authorizer.rb

```diff
@@ -206,13 +206,7 @@ def get_and_store_credentials_from_code options = {}
       #  Unique ID of the user for loading/storing credentials.
       def revoke_authorization user_id
         credentials = get_credentials user_id
-        if credentials
-          begin
-            @token_store.delete user_id
-          ensure
-            credentials.revoke!
-          end
-        end
```

Comment:
I refactored the lines above to a single line to solve a Rubocop complaint:

`lib/googleauth/user_authorizer.rb:54:5: C: Metrics/ClassLength: Class has too many lines. [113/110]`

Reply:
Honestly, we should prefer updating the rubocop config over doing this.

Reply:
Totally agree, happy to do that.

Reply:
Increased ClassLength to 200, removed previous exclusion.